pkg/operator/targetconfigcontroller/targetconfigcontroller_test.go

```diff
@@ -182,6 +182,92 @@ func Test_manageKubeSchedulerConfigMap_v311_00_to_latest(t *testing.T) {
 	}
 }
 
+// Added unit test for manageSchedulerKubeconfig
+func defaultKubeconfigConfigMap() *corev1.ConfigMap {
+	defaultKubeconfigCmYaml := []byte(`apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scheduler-kubeconfig
+  namespace: openshift-kube-scheduler
+data:
+  kubeconfig: |
+    apiVersion: v1
+    clusters:
+      - cluster:
+          certificate-authority: /etc/kubernetes/static-pod-resources/configmaps/serviceaccount-ca/ca-bundle.crt
+          server: https://127.0.0.1:443
+        name: lb-int
+    contexts:
+      - context:
+          cluster: lb-int
+          user: kube-scheduler
+        name: kube-scheduler
+    current-context: kube-scheduler
+    kind: Config
+    preferences: {}
+    users:
+      - name: kube-scheduler
+        user:
+          client-certificate: /etc/kubernetes/static-pod-certs/secrets/kube-scheduler-client-cert-key/tls.crt
+          client-key: /etc/kubernetes/static-pod-certs/secrets/kube-scheduler-client-cert-key/tls.key
+`)
+
+	curCM := &corev1.ConfigMap{}
+	if err := runtime.DecodeInto(codec, defaultKubeconfigCmYaml, curCM); err != nil {
+		return nil
+	}
+
+	return curCM
+}
+
+var configMapKubeConfigCMDefault = defaultKubeconfigConfigMap()
```

Comment:
do you need this function? it seems like you could just declare the configmap literal rather than starting with `byte[]` and decoding that to the configmap object

Reply:
Yes, why not? I updated the code with literal configmap.